incubator/logstash/templates/configmap.yaml

```diff
@@ -12,3 +12,6 @@ data:
   logstash.yml: |-
     http.host: "0.0.0.0"
     path.config: /usr/share/logstash/pipeline
+{{- if .Values.configData }}
+{{ toYaml .Values.configData | indent 4 }}
+{{- end }}
```

Comment:
Wouldn't it make sense to only use `configData` and put the other configs in there as defaults?

Reply:
My initial though is that the pipeline directory needs to stay as is because we are using that directory to mount the pipelines ConfigMap and if that was overridden it would stop working.
The http.host should also probably stay as-is as well.